packages/package_info/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -8,14 +8,14 @@
 
 /* Begin PBXBuildFile section */
```

Comment:
I'm not sure about these changes to pbxproj. cc @xster can you take a quick look?

Reply:
Looks ok. It's just updates on the xcode project being executed with a new version of cocoapods and with the recent app.flx -> flutter_assets changes